packages/app/cypress/e2e/integration/top-nav.spec.ts

```diff
@@ -1,4 +1,4 @@
-describe('App Top Nav Workflows', () => {
+describe('App Navigation', () => {
```

Comment:
What is App Top Nav vs App Navigation in this context? Only workflows form within the header bar?

Reply:
The intention is to be only workflows from within the header bar, yes. I was trying to align with the verbiage in the req spreadsheet, but thinking about it more, I'd consider 'navigation' to be the links on the side to go between Specs/Runs/Settings/etc.... This might be better renamed as 'header-bar.spec.ts'.

I will revert this specific change to keep things consistent. Let's get the additional test cases merged and follow up, a rename in this PR will cause some thrash with the existing test cases.